package.json

```diff
@@ -9,13 +9,16 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "firebase": "^9.16.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "uuidv4": "^6.2.13"
   },
   "devDependencies": {
     "@types/react": "^18.0.26",
     "@types/react-dom": "^18.0.9",
     "@vitejs/plugin-react": "^3.0.0",
+    "firebase-tools": "^11.21.0",
```

Comment:
I'm not sure if firebase-tools is necessary, I don't think we need to access the CLI from the frontend